### DISCUSSION TITLE
Ability to choose ROM size for Genesis/Mega Drive

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#962] [Ideas & Suggestions]
It seems a kind of big oversight - unless it is one on my part and I'm missing it - that we can't change the size of a MD cart dump. Sometimes the header is wrong. Once I figure it out, I submit the change to the MD.ino to detect that cart and dump at the right size, if possible (some carts don't have enough unique header info - in those cases I just update the ino to dump all carts at a certain size temporarily). All good once that is in the ino, but it would make determining the right size a lot easier, and allow those to dump that aren't into editing an ino, to get proper dumps, do overdumps to check size, etc.

Most of the other consoles have an ability to select a custom size, and many other options. With MD, it seems to be 'what the cart header says unless there's a fix in the ino already' - which can lead to me re-flashing my OSCR 5 or 6 times to make sure I'm getting the right size. A size choice from the menu would save a lot of re-flashing!

**Comment by sanni** [comment]
Added.
